tools/alara_output_processing/alara_output_processing.py

```diff
@@ -61,42 +170,83 @@ def _parse_table_data(
             current_table_lines,
             current_parameter,
             current_block
-        ):
-            '''
-            Parse a block of table lines with StringIO into an ALARADFrame and
-                store in the results dictionary.
-            
-            Arguments:
-                self (alara_output_processing.FileParser): FileParser object.
-                current_table_lines (list of str): Lines of the current table,
-                    each stored as a separate string.
-                results (dict): Dictionary that stores all parsed tables,
-                    keyed by parameter and block name.
-                current_parameter (str): Specific quantitative value
-                    represented in the table (e.g. specific activity, number
-                    density, etc.)
-                current_block (str): Block iterated upon in ALARA run.
-            Returns:
-                None
-            '''
-
-            adf = ALARADFrame(pd.read_csv(
-                StringIO('\n'.join(current_table_lines)), sep=r'\s+'
-            ))
-
-            adf.columns = [c.replace('_', '') for c in adf.columns]
-            key = f'{current_parameter} - {current_block}'
-            self.results[key] = adf
+    ):
+        '''
+        Parse a block of table lines with StringIO into a csv.DictReader and
+            store each row as a dictionary in a list of rows.
+        
+        Arguments:
+            self (alara_output_processing.FileParser): FileParser object.
+            current_table_lines (list of str): Lines of the current table,
+                each stored as a separate string.
+            results (dict): Dictionary that stores all parsed tables,
+                keyed by parameter and block name.
+            current_parameter (str): Specific quantitative value
+                represented in the table (e.g. specific activity, number
+                density, etc.)
+            current_block (str): Block iterated upon in ALARA run.
+        
+        Returns:
+            current_table_rows (list of dicts): List of each row in the
+                table's rows as a dictionary of the form:
+                row = {
+                    'time' : Cooling time of data entry in seconds,
+                    'nuclide' : Individual nuclide,
+                    'run_lbl' : Distinguisher between runs,
+                    'block' : ALARADFrame block integer enumerator,
+                    'block_num' : Geometric position of block,
+                    'variable' : ALARADFrame variable integer enumerator,
+                    'unit' : Unit for corresponding variable,
+                    'value' : Float value for the corresponding variable
+                } 
+        '''
+
+        # Prepare table text for csv.DictReader
+        text = '\n'.join(current_table_lines)
+        comma_delim = re.sub(r'\t+|\s{2,}', ',', text.strip())
```

Comment:
Is this maybe why you put the underscore in the times in the header, so that they don't match this regex?